.gitignore

```diff
@@ -32,3 +32,6 @@ powsybl.log
 
 # Generated readthedocs pages
 build-docs/
+
+# dotfile for environment variables
+.envrc
```

Comment:
Could you remove this from current PR? It's not related to current PR. 

Besides it seems that it's specific to your workflow. In that case you need to add this pattern in your `$GIT_DIR/info/exclude` or in your `~/.gitconfig` (`core.excludesFile` property). If not, feel free to open a PR to add it!